Include locale.h for LC_MESSAGE definition

This fixes build error
Source/WebCore/accessibility/atspi/AccessibilityRootAtspi.cpp:115:51: error: use of undeclared identifier 'LC_MESSAGES'
            return g_variant_new_string(setlocale(LC_MESSAGES, nullptr));
                                                  ^
Upstream-Status: Submitted [https://bugs.webkit.org/show_bug.cgi?id=239030]
Signed-off-by: Khem Raj <raj.khem@gmail.com>

--- a/Source/WebCore/accessibility/atspi/AccessibilityRootAtspi.cpp
+++ b/Source/WebCore/accessibility/atspi/AccessibilityRootAtspi.cpp
@@ -30,6 +30,7 @@
 #include "FrameView.h"
 #include "Page.h"
 #include <glib/gi18n-lib.h>
+#include <locale.h>
 
 namespace WebCore {
 
